### PR TITLE
[TA-873] Test validations for new Matter Types for Asylum and Immigration (hourly rates)

### LINKED ISSUE #86: Ensure the /tmp folder exists in the repository
When the repo is cloned and a test executed which uses a dynamically created bulkload file an error occurs.  This is because the code expects to place the bulkload file in /features/support/fixtures/bulkload/tmp.  This folder doesn't exist in the repo and needs to be initially created by the tester manually to avoid the error.

Easy solution would be to create the empty folder as a permanent fixture of the repo.

### PR DESCRIPTION
**Pull request by your**
## What does this pull request do?

Test validations for new Matter Types for Asylum and Immigration using hourly rates pricing scheme.

It also closes #86.

## Why make these changes?

It's a long story you can find in the [news](https://www.lawgazette.co.uk/news/government-u-turn-over-immigration-fee-reform/5105321.article). This is part of TA-872 / TA-873.

## Checklist

- [x] Provided link to a JIRA ticket: https://dsdmoj.atlassian.net/browse/TA-872, https://dsdmoj.atlassian.net/browse/TA-873

### CODE REVIEW

File: features/validations/legal_help/immigration_and_asylum/asylum_validations.feature [hunk 1]
```diff
@@ -26,19 +34,22 @@ Feature: Asylum Bulk load validations
 
   Scenario: Bulkload Civil Asylum outcomes with CASE_START_DATE around 25 October 2019
     The following Matter Types can be used from 25/10/2019.
-    IMCC:IGOL can only be used from 25/10/2019, even if an exemption code applies.
+    IAC[C-D]:IGOL can only be used from 25/10/2019, even if an exemption code applies.
```
Comment (solomonahmed) [inline features/validations/legal_help/immigration_and_asylum/asylum_validations.feature:37]: (v) minor suggestion?
reword ',even if an exemption code applies'
', and cannot be used with the Trafficking code TR001'

Comment (solomonahmed) [reply features/validations/legal_help/immigration_and_asylum/asylum_validations.feature:37]: Actually I think a test below has the working already needed:-
'Even if the trafficking criteria exemption code applies, this combination should not be valid.'?

**Review by solomonahmed — APPROVED** [review]
LGTM.
Minor wording issue.  But happy to approve.
Great work.

File: features/validations/legal_help/immigration_and_asylum/asylum_validations.feature (hunk 1 above)
Comment (your) [reply features/validations/legal_help/immigration_and_asylum/asylum_validations.feature]: @solomonahmed nice catch, done with b5840c4

**Pull request merged**